### ISSUE TITLE
not ripping reddit usernames with underscore suffix properly

### ISSUE THREAD

**Issue by Wololololo** [4pr0n/ripme#137]
I've noticed that one reddit user I've been ripping with an underscore suffix (user/enantiodromia_) does not rip properly. Ripme ends up ignoring the underscore suffix and ends up ripping u/enantiodromia instead(5 gigs of some ESO screenshot garbage). 

When you rip the username with the trailing underscore, it looks as if it rips that user properly, however, the file directory its saved to, and the username in the history log seems to leave out the underscore. This results in RipMe trying to rip a different user when you try to re-rip from history.
EDIT: it doesn't look like its ripping the full album of "user/enantiodromia_" and ends up ripping "user/enantiodromia"

I'd like to test that out on other users but I'm not too familiar with other usernames with the trailing underscore.
Can this be looked at and remedied?

tldr;  RipMe does not properly create directories of usernames with trailing underscores. This is also reflected in the history when trying to re-rip from history.

**Comment by 4pr0n** [comment]
Just rip the `/submitted` URL for that user...

`http://www.reddit.com/user/enantiodromia/submitted`

That only shows posts made by the user and ignores comments.
